challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+   input.map { |e|  e.capitalize}
 end
 
 def fetch_the_dog(input)
   #implement your solution here
+  input.select{|e|  e=="dog"}
 end
 
 def no_dogs_allowed(input)
   #implement your solution here
+  input.reject{|e|  e=="dog"}
 end
 
 def count_the_animals(input)
   #implement your solution here
+  input.length
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input[0,2]
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.select{|e| e.match(/^[cd]/)}
```

Comment:
See if you can find a method in the Enumerables documentation that is even more apt than `select`.  (I think the rules are that you aren't supposed to repeat methods in here.)